### DISCUSSION TITLE
ESP Mini Pico Build

### DISCUSSION THREAD

**Discussion by CGameDev** [JK-Rolling/DuinoI2C_ESP#3] [Show and tell]
This mini Pico build was inspired by the design and schematic of JK-Rolling's DuinoI2C_ESP research and development and contribution towards the Duino Coin community.  This build has 10 Pico and 2 D1 Mini per board. The lights on the compartments are configured to the PCB board. There are Pins out for optional power, LCD, and a power switch if so choose.

**Mining Board Blue Print.**
![image](https://github.com/user-attachments/assets/a3245fd7-a597-45a2-9d5d-ab8c847f4724)

**Mining Board 3D Representation**
![image](https://github.com/user-attachments/assets/2f636ba7-0bbe-45ad-aba8-86b81a7879ec)

The Finish Product
![IMG_0499](https://github.com/user-attachments/assets/a505f52c-192e-470e-ba8a-363afdbc8635)

**Build Log** 
https://youtu.be/zEK8xWIKxjE?si=5A3_h3E5WLfbkaYd

**Comment by JK-Rolling** [comment]
awesome mining rig that is stable, over-engineered and robust!
Do let us know if you manage to setup shop!

**Comment by JK-Rolling** [comment]
I believe your online shop is ready. Do you mind the share the link here?

**Reply by CGameDev** [reply]
https://computeruniversaltechnologysystems.ca/duino-coin-product/
